docs/architecture/adr-023-data-fee-market.md

```diff
@@ -0,0 +1,60 @@
+# ADR 023: DA Fee Market
+
+## Status
+
+Proposed
+
+## Changelog
+
+- 2023/06/28: Initial draft
+
+## Introduction
+
+At it's core, the Celestia network can be described as a market for the commodity of published data. By outsourcing the publication of data to Celestia, a rollup can mitigate the possibility of equivocation that may cause different machines to execute on a different set and order of transactions resulting in different states. In the field of distributed systems, this is known as split-brain. Thus, the market encompasses buyers who are willing to pay various amounts on the publication of their data, and sellers who are willing to supply the machines to perform the computation, storage and networking as defined by the protocol. This document explores the various dynamics of this market and makes some preliminary decisions on how to best design the fee market.
```

Comment:
**definitely** not blocking, but I do think we could get away with distilling this

```suggestion
At it's core, the Celestia network can be described as a market for the commodity of published data. This document explores the various dynamics of this market and makes some preliminary decisions on how to best design the fee market.
```